src/Messages/OWSMessageSender.m

```diff
@@ -952,7 +995,7 @@ - (void)handleMessageSentLocally:(TSOutgoingMessage *)message
 
 - (void)handleMessageSentRemotely:(TSOutgoingMessage *)message sentAt:(uint64_t)sentAt
 {
-    [self saveMessage:message withState:TSOutgoingMessageStateDelivered];
+    [message updateWithWasDelivered];
```

Comment:
Not your bug, but shouldn't this actually just be "sent"?

We have no way of knowing if it was delivered to Bob yet, right?